model/schema/core.yaml

```diff
@@ -426,11 +454,20 @@ classes:
       These should typically have an NCBI curie.
     slots:
       - specimen_genomic_model
+      - cross_references
     slot_usage:
       specimen_genomic_model:
         description: >-
           The strain or genotype of origin for the genome sequence.
 
+  GenomeAssemblyDTO:
+    is_a: BiologicalEntityDTO
+    description: >-
+      Ingest class for genome assemblies
+    slots:
+      - specimen_genomic_model_curie
```

Comment:
The new paradigm for DTO slots referring to classes inheriting from SubmittedObject is to use _identifier as the suffix rather than _curie

Reply:
Fixed